glue/sales_val_flagging.py

```diff
@@ -102,45 +102,42 @@ def ptax_adjustment(df, groups, ptax_sd, condos: bool):
     group_string = "_".join(groups)
 
     if not condos:
-        df["ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
+        df["sv_ind_ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
             (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
             | (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
             | (df[f"sv_price_per_sqft_deviation_{group_string}"] >= ptax_sd[1])
             | (df[f"sv_price_per_sqft_deviation_{group_string}"] <= -ptax_sd[0])
         )
 
-        # Determine the ptax direction
-        conditions = [
-            (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
-            | (df[f"sv_price_per_sqft_deviation_{group_string}"] >= ptax_sd[1]),
-            (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
-            | (df[f"sv_price_per_sqft_deviation_{group_string}"] <= -ptax_sd[0]),
-        ]
     else:
-        df["ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
+        df["sv_ind_ptax_flag_w_deviation"] = df["ptax_flag_original"] & (
             (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1])
             | (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0])
         )
 
-        # Determine the ptax direction
-        conditions = [
-            (df[f"sv_price_deviation_{group_string}"] >= ptax_sd[1]),
-            (df[f"sv_price_deviation_{group_string}"] <= -ptax_sd[0]),
-        ]
+    df["sv_ind_ptax_flag_w_deviation"] = df["sv_ind_ptax_flag_w_deviation"].astype(int)
 
-    directions = ["High", "Low"]
-    df["ptax_direction"] = np.select(conditions, directions, default=np.nan)
+    # Adjust order to make classify_outliers function smoother
```

Comment:
**[Nitpick, non-blocking]** A small recommended change to this comment, since I misinterpreted it on first read:

```suggestion
    # Adjust order of indicator columns to make classify_outliers function smoother
```

But also, per my comment below, perhaps there's a way to better define priority such that the order of these columns is not important.